### PR TITLE
docs: ✏️ add browser support doc (#trivial)

### PR DESCRIPTION
**Pull request by kaisermann**
#### What is the purpose of this pull request?

Add initial browser support documentation

Related to: #14

### CODE REVIEW

File: docs/Browser Support.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- [`Object.keys()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)
+- [`Object.values()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values)
+- [`Promise()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)
+- [`Set()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)
+- [`String.prototype.endsWith()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith)
+- [`String.prototype.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes)
+- [`String.prototype.startsWith()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith)
+- [`Symbol()`](https://developer.mozilla.org/en-US/docs/Glossary/Symbol)
+- [`URL()`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL)
+- [`WeakMap()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap)
```
Comment (salesfelipe) [inline docs/Browser Support.md]: Missing `Math.trunc`, like: https://github.com/vtex/checkout-confirmation-ui/pull/119

Comment (kaisermann) [reply docs/Browser Support.md]: We don't add it automatically 😢 : https://github.com/vtex/render-server/blob/master/node/utils/assetLoader.ts#L27

File: README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ Main repository for VTEX's javascript standards. Issues should be used to start
 
 ```tree
 docs/                         # standards documentation
+├── Browser\ Support.md       # vtex browser support documentation
```
[the remark this answers is not in the document]
Comment (kaisermann) [reply README.md]: 💅

File: docs/Browser Support.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- [MDN - Mozilla Developer Network](https://developer.mozilla.org/)
+- [Can I Use](https://caniuse.com/)
+
+If the functionality is not well supported and yet it's needed for some _important_ reason, a [polyfill or ponyfill](https://ponyfoo.com/articles/polyfills-or-ponyfills) can help solving the problem. Keep in mind that polyfills add extra weight to the store assets and not always perform as good as their native implementations.
+
+## Polyfills
+
+> A polyfill is a piece of code used to provide modern functionality on older browsers that do not natively support it.
+
+To be possible to write and use modern Javascript and CSS features while supporting all the way to IE 11, some polyfills are automatically added to every VTEX store:
```
Comment (klzns) [inline docs/Browser Support.md]:
```suggestion
To be possible to write and use modern Javascript and CSS features while supporting all the way to IE 11, some polyfills are automatically added **only** to VTEX IO stores:
```

File: docs/Browser Support.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+<!-- code_chunk_output -->
+
+- [Guidelines](#guidelines)
+- [Polyfills](#polyfills)
+
+<!-- /code_chunk_output -->
+
+VTEX browser support is split in two contexts: `storefront` and `admin areas`.
+
+For `storefronts`, we strive to support all major browsers all the way to IE 11:
```
Comment (klzns) [inline docs/Browser Support.md]: Maybe we should add the we guarantee that the store **works**. The layout may vary depending on the browser due to CSS browser support.

File: README.md [hunk 5]
```diff
@@ -6,6 +6,7 @@ Main repository for VTEX's javascript standards. Issues should be used to start
 
 ```tree
 docs/                         # standards documentation
+└── Browser\ Support.md       # vtex browser support documentation
```
Comment (kevinch) [inline README.md]:
```suggestion
└── Browser\ Support.md       # VTEX browser support documentation
```

File: docs/Browser Support.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+Edge 17+
+Safari 8+
+Chrome 77+
+Firefox 70+
+```
+
+For `admin areas`, the main targets are evergreen browsers, such as Firefox, Chrome, the new Edge, etc:
+
+```text
+# Browserslist example (date 29/01/2020)
```
Comment (kevinch) [inline docs/Browser Support.md]: Nothing about IE11? Wondering If we could check already with analytics the use of the admin by store owners.

File: docs/Browser Support.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+Edge 17+
+Safari 12.1
+Chrome 77+
+Firefox 70+
+```
+
+## Guidelines
+
+It's usually in a dev's heart the desire to use the newest Javascript feature or a new fancy CSS property, such as `display: grid`, `position: sticky`, `Map()`, `Set()`, etc. However, _browser support_ and _cross-browser compatibility_ are the first concerns that should come to the developer's mind. Make sure to always check `MDN` and `Can I Use` for the current support of the desired functionality.
```
Comment (kevinch) [inline docs/Browser Support.md]:
```suggestion
It's usually in a dev's heart the desire to use the newest Javascript feature or a new fancy CSS property, such as `display: grid`, `position: sticky`, `Map()`, `Set()`, etc. However, _browser support_ and _cross-browser compatibility_ are the first concerns that have to come to the developer's mind. Make sure to always check `MDN` and `Can I Use` for the current support of the desired functionality.
```

File: docs/Browser Support.md (hunk 6 above)
Comment (kaisermann) [reply docs/Browser Support.md]: That would be great indeed! Let's move this to the slack channel

File: docs/Browser Support.md (hunk 4 above)
Comment (kaisermann) [reply docs/Browser Support.md]: Done!

File: docs/Browser Support.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+- [Polyfills](#polyfills)
+
+<!-- /code_chunk_output -->
+
+VTEX browser support is split in two contexts: `storefront` and `admin areas`.
+
+For `storefronts`, we strive to support all major browsers all the way to IE 11:
+
+```text
+# Browserslist example (date 29/01/2020)
```
Comment (klzns) [inline docs/Browser Support.md]:
```suggestion
# Browserslist example (date 2020-01-29)
```

File: docs/Browser Support.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+Chrome 77+
+Firefox 70+
+```
+
+_Note: even though we support older browsers such as IE 11 and Safari 8, the layout may slightly vary due to CSS browser support._
+
+For `admin areas`, the main targets are evergreen browsers, such as Firefox, Chrome, the new Edge, etc:
+
+```text
+# Browserslist example (date 29/01/2020)
```
Comment (klzns) [inline docs/Browser Support.md]:
```suggestion
# Browserslist example (date 2020-01-29)
```

**Pull request merged**